src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -127,7 +127,11 @@ export const SearchResults = ({
 
   if (fetching) {
     return (
-      <va-loading-indicator set-focus message="Loading search results..." />
+      <va-loading-indicator
+        set-focus
+        message="Loading search results..."
+        uswds
```

Comment:
I don't think this component has a v3 version: https://design.va.gov/storybook/?path=/docs/components-va-loading-indicator--default

Reply:
Good catch! I'll remove that shortly!